onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -781,7 +781,65 @@ def is_safe_to_fuse_nodes(self, nodes_to_remove, keep_outputs, input_name_to_nod
                             return False
         return True
 
+    def graph_topological_sort(self, graph):
```

Comment:
This function could be a static method and called in the topological_sort function of quantization.